src/common/mailing.ts

```diff
@@ -46,6 +52,52 @@ export const baseNotificationEmailData: SendEmailRequestOptionalOptions = {
   queue_draft: false,
 };
 
+const isSubscribed = (
+  subs: { topics?: Record<string, boolean> },
+  topic: string,
+): boolean => !(subs?.topics?.[`topic_${topic}`] === false);
+
+export const resubscribeUser = async (
+  cio: TrackClient,
+  userId: string,
+): Promise<void> => {
+  if (!process.env.CIO_APP_KEY) {
+    return;
+  }
+
+  await cio.identify(userId, { unsubscribed: false });
+};
+
+export const syncSubscription = async function (
+  userId: string,
+  con: DataSource,
+): Promise<void> {
+  if (!process.env.CIO_APP_KEY) {
+    return;
+  }
+
+  const atts = await cioApi.getAttributes(userId);
+  const subs = JSON.parse(
+    atts?.customer?.attributes?.cio_subscription_preferences || '{}',
+  );
+  const marketing = isSubscribed(subs, '4');
+  const notifications = isSubscribed(subs, '7');
+  const digest = isSubscribed(subs, '8');
```

Comment:
Not blocking but would be great if we can ENUM these values somewere?
I envision we'll do other stuff with it in the future and magic numbers are not super clear.

Reply:
done